cmd/vc-rest/startcmd/start.go

```diff
@@ -151,19 +131,72 @@ func createStartCmd(srv server) *cobra.Command {
 				hostURLExternal:      hostURLExternal,
 				universalResolverURL: universalResolverURL,
 				mode:                 mode,
+				localStorePath:       localStorePath,
 			}
+
 			return startEdgeService(parameters)
 		},
 	}
 }
 
+func getVCRestParameters(cmd *cobra.Command) (string, string, string, string, string, string, string, error) {
```

Comment:
returning more than 2/3 values is a sign they should be grouped in a struct instead.

Reply:
you need to change this method

Reply:
Oh, yes you're right. In fact, there's already a struct called vcRestParameters that's a perfect fit already. In hindsight it seems obvious...not sure why I missed that before. I've updated the code.